files/jira.rb

```diff
@@ -32,9 +49,18 @@ def create_issue(summary, full_description, project)
             ]
           }
         }
-        issue.save(issue_json)
-        url = get_options[:site] + '/browse/' + issue.key
-        puts "Created issue #{issue.key} at #{url}"
+        # FIXME - would this work
+        components = []
+        if components.size?
+          issue_json['fields']['components'] = components
```

Comment:
Gotcha. I'm a little worried about scope creep, @jaxxstorm mentioned that his company adds arbitrary fields all the time. I'm just worried about this handler getting more and more complex over time.